test/performace_check/test_fftw.py

```diff
@@ -13,30 +12,16 @@
 
 @profile
```

Comment:
See above comment about unused variables and using `_` instead. Otherwise this code will not measure anything and it'll look as if the ffts are blindingly fast :)